### PR TITLE
added V4.1 fixes #175

### LINKED ISSUE #175: ARTIC V4.1 primer scheme for Omicron
We might need to add support for V4.1 soon:

https://twitter.com/Scalene/status/1466070415986544640

https://community.artic.network/t/sars-cov-2-v4-1-update-for-omicron-variant/342

### PR DESCRIPTION
**Pull request by MarieLataretu**
fixes #175 

I avoided a `.` in the folder's name, since it could have an effect, when `simpleName` is used.

If you prefer something else, let me know!

### CODE REVIEW

**Comment by hoelzer** [comment]
but can I already select this primer scheme now, @MarieLataretu ? via `--primerV v4-1` ? 

And then we should likely also add the support of this kit to the `--help`

**Comment by hoelzer** [comment]
just performing a testrun w/ this setting now (although the primer kit originally used was V4)

**Comment by hoelzer** [comment]
besides, run through @MarieLataretu ! nice. If the first V4.1 sequencing pops up we can try this and if it works fine merge

File: poreCov.nf [hunk 1]
```diff
@@ -466,12 +466,12 @@ ${c_yellow}Parameters - Basecalling  (optional)${c_reset}
 
 ${c_yellow}Parameters - SARS-CoV-2 genome reconstruction (optional)${c_reset}
     --primerV       Supported primer variants - choose one [default: ${params.primerV}]
-                        ${c_dim}ARTIC:${c_reset} V1, V2, V3, V4 
+                        ${c_dim}ARTIC:${c_reset} V1, V2, V3, V4, V4.1
```
Comment (hoelzer) [inline poreCov.nf:469]: but here I have to use V4-1, or? as the parameter

Comment (MarieLataretu) [reply poreCov.nf:469]: I renamed it. I typed V4-1 and it was ugly...

Comment (hoelzer) [reply poreCov.nf:469]: :D okay so now one can use `--primerV V4.1` ?

Comment (MarieLataretu) [reply poreCov.nf:469]: I checked - `{params.primerV}` is always used as it is

**Comment by MarieLataretu** [comment]
I updated the readme and help - also with `V4` :)

---
```
nextflow run poreCov.nf -profile singularity,local,test_fastq,stub --cores 2 --max_cores 2 --primerV V4.1
```
works fine, but I cheated with stub.

File: poreCov.nf (hunk 1 above)
Comment (MarieLataretu) [reply poreCov.nf:469]: Yes, `--primerV V4.1` should work :)

**Comment by hoelzer** [comment]
worked and looks good! amplicon recovered via V4.1 in omicron! I will merge, but once the ARTIC network puts the "offical" BED in their repository we might want to update again

thx!

**Pull request merged**